js/src/components/contact-information/store-address-card.js

```diff
@@ -66,7 +70,10 @@ const StoreAddressCard = ( { showValidation = false } ) => {
 	}
 
 	const handleRefreshClick = () => {
-		refetch();
+		setSaving( true );
+		updateGoogleMCContactInformation()
+			.then( refetch )
```

Comment:
Adding this refetch is unnecessary, because `updateGoogleMCContactInformation()` returns the new data and updates the data store.

Reply:
👍